docs/source/markdown/podman-artifact-add.1.md

```diff
@@ -0,0 +1,48 @@
+% podman-artifact-add 1
+
+## WARNING: Experimental command
+*This command is considered experimental and still in development. Inputs, options, and outputs are all
+subject to change.*
+
+## NAME
+podman\-artifact\-add - Add an OCI artifact to the local store
+
+## SYNOPSIS
+**podman artifact add** [*options*] *name* *file* [*file*]...
```

Comment:
Try this? Since there's no options other than help, it should [skip](https://github.com/containers/podman/blob/9f1fee2a0b119024750975e2b16c1a89edc615d9/test/system/600-completion.bats#L84) the completion test. 

```suggestion
**podman artifact add** *name* *file* [*file*]... 
```

Reply:
that was the cause.  @baude just make sure to change it in the .go file as well (inspect and rm need the same tweak)